### PR TITLE
Stop sending SMSes on whatsapp send errors

### PR DESCRIPTION
**Pull request by rudigiesler**

### CODE REVIEW

File: scripts/vaccine-upload/vaccine-upload.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
+from urllib.parse import urljoin
+
+import requests
+
+MSISDN_FILENAME = "users.csv"
+TURN_URL = "https://whatsapp.turn.io/"
+TURN_TOKEN = (
```
Comment (erikh360) [inline scripts/vaccine-upload/vaccine-upload.py]: Not sure if you wanted to commit this?

**Review by erikh360 — COMMENTED** [review]
One comment and I see the tests are failing

File: scripts/vaccine-upload/vaccine-upload.py (hunk 1 above)
Comment (rudigiesler) [reply scripts/vaccine-upload/vaccine-upload.py]: Yeah thanks, was a mistake. Let me remove it and revoke the token

**Comment by rudigiesler** [comment]
@erikh360 Ready for another review

**Review by erikh360 — APPROVED** [review]

**Pull request merged**